packages/executor/src/execution/__tests__/variables-test.ts

```diff
@@ -29,13 +31,30 @@ const TestComplexScalar = new GraphQLScalarType({
   },
 });
 
+const TestFaultyScalarGraphQLError = new GraphQLError('FaultyScalarErrorMessage', {
```

Comment:
```suggestion
const TestFaultyScalarGraphQLError = createGraphQLError('FaultyScalarErrorMessage', {
```
This will fix the failing tests in v15